ducktape/command_line/main.py

```diff
@@ -56,6 +57,7 @@ def parse_args():
                              "This is primarily useful for test developers who want to interact with running " +
                              "services after a test has run. ")
     parser.add_argument("--version", action="store_true", help="display version")
+    parser.add_argument("--inject", action="store", help="Inject these arguments into the specified test. Use JSON to format.")
```

Comment:
Also, can we change the command line argument name? I know you use `inject` in the implementation of parameterization, but I don't think it's very intuitive for users. `--parameters` maybe, which directly mirrors the naming of the annotation?